fiftyone/operators/registry.py

```diff
@@ -90,9 +96,11 @@ def list_operators(self, include_builtin=True, type=None):
         for pctx in self.plugin_contexts:
             operators.extend(pctx.instances)
 
-        if include_builtin:
-            operators.extend(BUILTIN_OPERATORS)
-            operators.extend(BUILTIN_PANELS)
+        if not include_builtin:
+            operators = [op for op in operators if op._builtin is False]
```

Comment:
Instead of making it a property, can we just have a hardcoded list so that people can’t just mark their own as builtin?

Reply:
I think its impossible to prevent hijacking in this way. We would need some other mechanisms to fully prevent that. Either way lets first get the new structure in place then we can improve on the builtin mechanism.